doctr/models/recognition/zoo.py

```diff
@@ -49,6 +50,26 @@ def crnn_vgg16_bn_predictor(pretrained: bool = False, **kwargs: Any) -> Recognit
     return _predictor('crnn_vgg16_bn_predictor', pretrained, **kwargs)
 
 
+def crnn_resnet31_predictor(pretrained: bool = False, **kwargs: Any) -> RecognitionPredictor:
+    """Text detection architecture using a CRNN with a ResNet-31 backbone.
+
+    Example::
+        >>> import numpy as np
+        >>> from doctr.models import crnn_resnet31_bn_predictor
+        >>> model = crnn_resnet31_predictor(pretrained=False)
+        >>> input_page = (255 * np.random.rand(512, 128, 3)).astype(np.uint8)
```

Comment:
I didn't pay attention earlier to the shape of the mock page, but we should make it more reasonable :sweat_smile: